src/gui/tray/ActivityItem.qml

```diff
@@ -26,6 +27,10 @@ MouseArea {
     Accessible.name: (model.path !== "" && model.displayPath !== "") ? qsTr("Open %1 locally").arg(model.displayPath) : model.message
     Accessible.onPressAction: root.clicked()
 
+    function showReplyOptions(activityIndex) {
```

Comment:
can you remove the argument that is not used inside the function ?

Reply:
done